src/main/java/Dude.java

```diff
@@ -0,0 +1,159 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+// Base Task class
+abstract class Task {
+    String description;
+    boolean isDone;
```

Comment:
Maybe could private these variable or limited access of this variable?
I noticed the same issues in other places too.